frontend/Panel.js

```diff
@@ -295,6 +295,7 @@ class Panel extends React.Component {
         }}
         extraPanes={extraPanes}
         extraTabs={extraTabs}
+        onViewElementSource={(id, source) => this.viewElementSource(id, source)}
```

Comment:
Yes, it should be passed only if `showElementSource` prop is defined.
Maybe I should add flow-types to `PropState` component props and for `decorate` function?